frontend/src/component/PublicProfile/CustomFormManager.styl

```diff
@@ -0,0 +1,26 @@
+.CustomFormManager
+  position relative
+  &__title
+    margin-bottom 15px
+    font-size 24px
```

Comment:
Is there an existing constant font size value that could be used there? if not, should we add one?

Reply:
fontSizeTitle

Reply:
will fix